moveit_ros/planning/CMakeLists.txt

```diff
@@ -31,7 +38,7 @@ set(THIS_PACKAGE_INCLUDE_DIRS
     # kinematics_plugin_loader/include
     # robot_model_loader/include
     # constraint_sampler_manager_loader/include
-    planning_pipeline/include
+    # planning_pipeline/include
```

Comment:
Doesn't planning_pipeline compile?

Reply:
Yes, it compiles. Previously I tried to disable unrelated components. Added it back.